user_guide_src/source/concepts/factories.rst

```diff
@@ -88,13 +140,13 @@ Configurations
 
 To set default component options, create a new Config files at **app/Config/Factory.php**
 that supplies options as an array property that matches the name of the component. For example,
-if you wanted to ensure that all Filters used by your app were valid framework instances,
-your **Factory.php** file might look like this:
+if you wanted to ensure that all **Filters** used by your app were valid framework instances,
+your **app/Config/Factory.php** file might look like this:
 
 .. literalinclude:: factories/005.php
 
 This would prevent conflict of an unrelated third-party module which happened to have an
```

Comment:
Drop this "unrelated" in favor of the one in the next line.

Reply:
Done.